changelog/chore-e2e-test-report-template

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: Create e2e test report issue template.
```

Comment:
I think we don't need this changelog entry to be pulled into release notes (changelog). IIRC there is a recommended way of skipping the changelog entry

Reply:
This is the recommended way, it won't be pulled into the changelog 👍

Reply:
TIL, thanks!